vxquery-core/src/main/java/org/apache/vxquery/compiler/algebricks/VXQueryExpressionRuntimeProvider.java

```diff
@@ -145,4 +145,4 @@ public IUnnestingEvaluatorFactory createUnnestingFunctionFactory(UnnestingFuncti
             throw new AlgebricksException(e);
         }
     }
-}
```

Comment:
I'm going to stop commenting this (looks like a lot of files). I'm not sure whether this needs to be changed, but if so, it needs to for all these files.